FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/JobConfigurationConstants.cs

```diff
@@ -70,5 +70,9 @@ public static class JobConfigurationConstants
         /// The running job count threshold to check running job complete
         /// </summary>
         public const int CheckRunningJobCompleteRunningJobCountThreshold = 5;
+
+        public const int DefaultInitialDicomStartOffset = 0;
```

Comment:
how about creating a new file for Dicom constants? as this file is for common job configuration.

Reply:
The file stores constants for FHIR as well, such as `DefaultNumberOfPatientsPerProcessingJob`, `DefaultInitialOrchestrationIntervalInSeconds`, `DefaultIncrementalOrchestrationIntervalInSeconds`. We can separate them when refining job management relevant classes if needed.